projects/pic-shocks/injector.py

```diff
@@ -179,6 +179,9 @@ def damp_em_fields(self, grid, lap, conf):
             elif conf.twoD:
                 i,j = ind
                 k = 0
+            elif conf.oneD:
```

Comment:
better to re-order these as

```
if conf.oneD:

elif conf.twoD:

else conf.threeD:
```

Then we follow 1D/2D/3D logic. In addition, the last `else` ensures that something is always selected.